code/style.css

```diff
@@ -1,4 +1,213 @@
+
+/* Universal styles */
+
 html {
-  background: #0025ff;
-  color: #fff;
+  box-sizing: border-box;
+  font-family: 'Work sans', Helvetica, sans-serif;
+  font-size: 16px;
+}
+
+body {
+  margin: 0;
+  background-color: #9DC88D;
+}
+
+h1, h2 {
+  font-family: 'Neuton', Garamond, serif;
+}
+
+p {
+  font-family: 'Work sans', Helvetica, sans-serif;
+  line-height: 1.4;
+}
+
+a {
+  font-family: 'Work sans', Helvetica, sans-serif;
+  font-weight: 700;
+}
+
+/* Header content */
+
+.header {
+  height: 5rem;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  padding: 1rem 0;
+  background-color: #164A41;
+  color: #FFFFFF;
+}
+
+.logo-container {
+  padding: 1rem;
+}
+
+.header h1 {
+  font-size: 2.5rem;
+}
+
+.header-nav {
+  display: none;
+  /* The nav is not displayed in mobile devices*/
+}
+
+.header-nav li {
+  display: inline;
+}
+
+.header-nav a {
+  text-decoration: none;
+  letter-spacing: 0.08rem;
+  padding: 0 0.5rem;
+  color: #F1B24A;
+}
+
+.header-nav a:hover {
+  text-decoration: underline;
+}
+
+.main-container {
+  max-width: 1000px;
+  margin: 0 auto;
+  /*Centers the main container*/
+}
+
+/* Big news section */
+
+.big-news-section {
+  display: grid;
+  grid-template-rows: 1fr auto;
+  grid-template-columns: 1fr;
+  gap: 0;
+  background-color: #4D774E;
+  color: #FFFFFF;
+}
+
+.big-news-section img {
+  max-width: 100%;
+  height: auto;
+}
+
+.big-news-image-container {
+  padding: 1rem;
+}
+
+.big-news-article {
+  padding: 1rem;
+}
+
+.big-news-section h2 {
+  margin: 1rem 0;
+}
+
+.big-news-section p {
+  font-size: 0.8rem;
+}
+
+/* Other news section */
+
+.other-news-section {
+  margin: 1rem;
+}
+
+.other-news-card {
+  padding: 1rem;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+  background-color: #FFFFFF;
+  margin-bottom: 1rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.other-news-card:hover {
+  background-color: #9DC88D;
+}
+
+.other-news-card img {
+  max-width: 100%;
+  height: auto;
+}
+
+.other-news-text {
+  text-align: center;
+}
+
+.other-news-text p {
+  font-size: 0.8rem;
+  font-weight: 200;
+}
+
+/* Footer */
+
+footer {
+  background-color: #164A41;
+  color: #F1B24A;
+  height: 4rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 0.7rem;
+}
+
+/* Tablet media queries */
+
+@media only screen and (min-width: 768px) and (max-width: 992px) { 
+  .header-nav {
+    display: block;
+    /*This changes the order of the items in the header*/
+    order: -1;
+  }
+
+  .header {
+    justify-content: space-between;
+  }
+
+  /*Hides the header h1*/
+  .header h1 {
+    display: none;
+  }
```

Comment:
I would like to see the name of the page for all different screen sizes. That was something I missed when looking at the page for the tablet size.

Reply:
Yes, I was thinking about that too... I'll change it.